src/Raven.Client/Documents/Session/Loaders/MultiLoaderWithInclude.cs

```diff
@@ -67,14 +68,52 @@ public ILoaderWithInclude<T> Include<TInclude>(Expression<Func<T, IEnumerable<st
             return Include(IncludesUtil.GetPrefixedIncludePath<TInclude>(path.ToPropertyPath(), _session.Conventions));
         }
 
+        /// <summary>
+        /// Includes a counter of the document to be loaded, by counter name
+        /// <param name="name">Name of the counter to include.</param>
+        /// </summary>
+        public ILoaderWithInclude<T> IncludeCounter(string name)
+        {
```

Comment:
assert if name is not null or whitespace?

Reply:
done